units/patch_intent.go

```diff
@@ -955,6 +955,18 @@ func (j *patchIntentProcessor) buildGithubPatchDoc(ctx context.Context, patchDoc
 			"intent_id":   j.IntentID,
 		}))
 		return false, err
+	} else if !isMember {
+		grip.Debug(message.Fields{
```

Comment:
Is this intended to be a short term log?

Reply:
Long term -- mentioned this in the PR comment but I think generally this log is helpful for future reference, since we occasionally hit unauthorized errors and its helpful to know a bit more about why so we can communicate with the user. That said, I could attach it specifically to a ticket if we'd like to keep the logs clear, which I can see an argument for (maybe could just be removed after [DEVPROD-707](https://jira.mongodb.org/browse/DEVPROD-707))

Reply:
If we think [DEVPROD-707](https://jira.mongodb.org/browse/DEVPROD-707)'s completion could be reason to remove this log then it'd be helpful to add this as either a comment or a ticket field in the log itself, just so we can remember

Reply:
I'll add a comment, since we still might want to hold onto this log.